public/modules/custom/grants_profile/src/Form/ModalApplicationOfficialForm.php

```diff
@@ -76,12 +76,11 @@ public function getFormId(): string {
   public static function getOfficialRoles(): array {
     return [
       1 => t('Chairperson'),
-      2 => t('Contact person'),
-      3 => t('Other'),
-      4 => t('Financial officer'),
-      5 => t('Auditor'),
-      7 => t('Secretary'),
-      8 => t('Vice Chairperson'),
+      2 => t('Financial officer'),
```

Comment:
Nämä olivat mun mielestä Modalissa juuri oikein päin, eli niitä ei pidä muuttaa, jos käyttää tätä modalin versiota.